prow/cmd/deck/pr_history_test.go

```diff
@@ -291,7 +299,7 @@ func TestGetPRBuildData(t *testing.T) {
 				jobName:      "build-snowman",
 				index:        0,
 				ID:           "456",
-				SpyglassLink: "/view/gcs/chum-bucket/pr-logs/pull/123/build-snowman/456",
+				SpyglassLink: "/view/gs/chum-bucket/pr-logs/pull/123/build-snowman/456",
```

Comment:
We have to keep supporting the old urls because of compatibility reasons

Reply:
Yup agree. What changes in this test is not really what urls we're supporting. In this cases we're using the same `testBucket` as before and we're also getting the PR build data of the same jobs. 

What changes is the generated spyglass links. So from now on we would generate spyglass links with `gs` instead of `gcs`. Doesn't mean if somebody still has an old gcs link that it won't work. 

Of course this assumes that it's okay that after this PR we would generate links with `gs` instead of `gcs`